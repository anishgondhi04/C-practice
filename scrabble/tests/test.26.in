10
3
2 2 V hello
3 1 V hello
2 2 H hello
1
2 l 5

