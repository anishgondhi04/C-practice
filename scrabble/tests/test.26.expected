Invalid word placement: (3,1) V, hello
..........
..........
..hello...
..e.......
..l.......
..l.......
..o.......
..........
..........
..........
Place vertically at (4,2)
Place vertically at (5,2)
Place horizontally at (2,4)
Place horizontally at (2,5)
